src/generator.cs

```diff
@@ -2300,7 +2300,10 @@ public void Go ()
 					if (AttributeManager.HasAttribute<CoreImageFilterPropertyAttribute> (pi))
 						continue;
 
-					if (AttributeManager.HasAttribute<WrapAttribute> (pi.GetGetMethod ()) || AttributeManager.HasAttribute<WrapAttribute> (pi.GetSetMethod ()))
+					// we ensure that we do not get a NRE in the case we forgot and ExportAttr in a property
+					var hasWrapGet = pi.GetGetMethod () != null && AttributeManager.HasAttribute<WrapAttribute> (pi.GetGetMethod ());
+					var hasWrapSet = pi.GetSetMethod () != null && AttributeManager.HasAttribute<WrapAttribute> (pi.GetSetMethod ());
```

Comment:
This two could have been added in the if, but made the code 'ugly' and hard to understand. I think it is nicer to be readable.

Reply:
Believe it or not, I had that and though it was a little noise. Will do :)